react-form-controls/my-app/src/RegistrationFormUncontrolled.js

```diff
@@ -0,0 +1,27 @@
+export default function RegistrationFormUncontrolled() {
+  function handleSubmit(e) {
+    e.preventDefault();
+    const form = document.querySelector('#form');
```

Comment:
No `querySelector`s! We don't query the DOM in react. You should be utilizing your `event` parameter